Brewfile.lock.json

```diff
@@ -2,40 +2,40 @@
   "entries": {
     "brew": {
       "clang-format": {
-        "version": "19.1.3",
+        "version": "19.1.4",
```

Comment:
Following instructions from `check-tooling-versions.sh`:

> clang-format version mismatch, expected: 19.1.3, but found: 19.1.4. Please run `make init` to update your local dev tools. This may actually upgrade to a newer version than what is currently recorded in the lockfile; if that happens, please commit the update to the lockfile as well.

Let me know if you don't want this change in this PR, but it was preventing me from committing with the pre-commit hook in place.